share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-neoverse_v1/spack.yaml

```diff
@@ -2,19 +2,14 @@ spack:
   view: false
 
   definitions:
-  - compiler_specs:
-    - gcc
-    - gettext
-
-  - compiler_target:
-    - '%gcc@7.3.1 target=aarch64'
-
   - optimized_configs:
-    - gromacs
+    - gromacs target=neoverse_v1
+    - gromacs target=neoverse_n1
     # - lammps
```

Comment:
It would be good to have commented out specs have a comment next to them on why they are commented out.

Reply:
I have removed all unnecessary commented packages.